oc23/upload/system/library/v2pagecache.php

```diff
@@ -245,6 +245,16 @@ public function Cacheable() {
         } 
         $svar=$this->GetSessionVar();
         // don't cache for logged in customers or affiliates
+		/**
+         * if data in _SESSION has a nested array
+         * that requires a another one check
+         */
+        $logged = false;
+        foreach ($svar as $sval) {
+            if (!empty($sval['customer_id']) || !empty($sval['affiliate_id'])) {
+                $logged = true;
+            }
+        }
         if( !empty($svar['customer_id']) || !empty($svar['affiliate_id'])) {
```

Comment:
if( !empty($svar['customer_id']) || !empty($svar['affiliate_id']) || $logged) {
            $this->cacheable=false;
            return $this->cacheable;
        }